README.md

```diff
@@ -1 +1,53 @@
-# CAPSTONE-3
\ No newline at end of file
+# Global Cryptocurrency
+This is a module 3 capstone project at Microverse. It entails a React-Redux application that fetches digital currency data from a coins API. The applica allows user to filter information by different types of digital coins.
+
+
+## Built With
+
+- Javascript
+- React, Redux
+- Webpack, Babel, React Testing Library
+- Styled Components
+
+## Live Demo
```

Comment:
- [ ] Hello and excellent work on the app but as you see a link to a deployed react app is a requirement

![Capture web_14-10-2022_11319_github com](https://user-images.githubusercontent.com/61562626/195826333-39824e1f-0645-41f3-8da5-5c1613c37799.jpeg)